examples/segmentation_training.py

```diff
@@ -0,0 +1,123 @@
+        gt = gt.where(gt != 255, torch.tensor(-1, device=gt.device))
+        intersection = (pred * gt).sum(dim=(1, 2)).float()
+        union = (pred + gt).sum(dim=(1, 2)).float() - intersection
+        iou = (intersection / union).cpu()
+        self.results.append({'batch_size': len(gt), 'iou': iou})
+
+    def compute_metrics(self, results):
+        total_iou = sum(item['iou'].sum() for item in results)
+        total_size = sum(item['batch_size'] for item in results)
+        return dict(iou=100 * total_iou / total_size)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Distributed Training')
+    parser.add_argument(
+        '--launcher',
+        choices=['none', 'pytorch', 'slurm', 'mpi'],
+        default='none',
+        help='job launcher')
+    parser.add_argument('--local_rank', type=int, default=0)
+
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+    num_classes = 21  # Modify to actual number of categories.
+    norm_cfg = dict(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
+    train_set = torchvision.datasets.VOCSegmentation(
+        'data/VOC2012',
+        image_set='train',
+        download=True,
+        transform=transforms.Compose([
+            transforms.Resize((520, 520)),
+            transforms.ToTensor(),
+            transforms.Normalize(**norm_cfg)
+        ]),
+        target_transform=transforms.Compose([
+            transforms.Resize((520, 520), interpolation=0),
+            transforms.Lambda(
+                lambda x: torch.tensor(np.array(x), dtype=torch.long))
+        ]))
+    valid_set = torchvision.datasets.VOCSegmentation(
+        'data/VOC2012',
+        image_set='val',
+        download=True,
+        transform=transforms.Compose([
+            transforms.Resize((520, 520)),
+            transforms.ToTensor(),
+            transforms.Normalize(**norm_cfg)
+        ]),
+        target_transform=transforms.Compose([
+            transforms.Resize((520, 520), interpolation=0),
+            transforms.Lambda(
+                lambda x: torch.tensor(np.array(x), dtype=torch.long))
+        ]))
+    train_dataloader = dict(
+        batch_size=8,
+        dataset=train_set,
+        sampler=dict(type='DefaultSampler', shuffle=True),
+        collate_fn=dict(type='default_collate'))
+    val_dataloader = dict(
+        batch_size=8,
+        dataset=valid_set,
+        sampler=dict(type='DefaultSampler', shuffle=False),
+        collate_fn=dict(type='default_collate'))
+    runner = Runner(
+        model=MMDeeplabV3(num_classes),
+        work_dir='./work_dir',
+        train_dataloader=train_dataloader,
+        optim_wrapper=dict(optimizer=dict(type=SGD, lr=0.01, momentum=0.9)),
+        train_cfg=dict(by_epoch=True, max_epochs=10, val_interval=1),
+        val_dataloader=val_dataloader,
+        val_cfg=dict(),
+        val_evaluator=dict(type=IoU),
+        launcher=args.launcher,
```

Comment:
```suggestion
        launcher=args.launcher,
        cfg=dict(model_wrapper='MMDistributedDataParallel', find_unused_parameters=True)
```

`find_unused_parameters` should be configured other an error will be raised in ddp-training